docs/book/compatiblity_matrix.md

```diff
@@ -1,14 +1,18 @@
 # Compatibility Matrix for vSphere CSI Driver
 
-- vSphere CSI driver is released with specific version of CSI sidecar containers which has the minimum kubernetes version requirement.
+- vSphere CSI driver is released with specific version of CSI sidecar containers which has the minimum and maximum kubernetes version requirement.
 
-| vSphere CSI Driver | Minimum Kubernetes Version |
-|--------------------|----------------------------|
-| v1.0.2             | v1.14.0                    |
-| v1.0.3             | v1.14.0                    |
-| v2.0.0             | v1.16.0                    |
-| v2.0.1             | v1.17.0                    |
-| v2.1.0             | v1.17.0                    |
+Note:
+
+- vmware supports the deprecated release of the driver until End Of Life date.
+
+| vSphere CSI Driver | Minimum Kubernetes Version | Maximum Kubernetes Version            | Deprecated | End of Life  |
+|--------------------|----------------------------|---------------------------------------| -----------|--------------|
+| [v1.0.2](./releases/v1.0.2.md)             | v1.14.0                    | v1.16.0       | Yes        | January 2021 |
+| [v1.0.3](./releases/v1.0.3.md)             | v1.14.0                    | v1.16.0       | Yes        | January 2022 |
```

Comment:
Can we add like, Deprecation start date
Also, for 2.0.0 we can add deprecation? or mention that it will be deprecated soon (may be with a date as well) to discourage customers from picking it.

Reply:
updated page as per internal discussion. For now, I have not added the deprecation start date.